src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl

```diff
@@ -90,18 +90,24 @@ echo "Running '{{ PackageInstallCommand }}'..."
 echo
 {{ PackageInstallCommand }}
 
-{{ if NpmRunBuildCommand | IsNotBlank }}
-echo
-echo "Running '{{ NpmRunBuildCommand }}'..."
-echo
-{{ NpmRunBuildCommand }}
-{{ end }}
+{{ if CustomNpmRunBuildCommand | IsNotBlank }}
```

Comment:
so we are assuming (as per design) we will ignore npm run build if custom npm run build command is enabled

Reply:
yes that's right